### PR TITLE
Shindo to Minitest: image, planning

### PR DESCRIPTION
**Pull request by gildub**
Migrates image and planning from Shindo to Minitest
Note: This is not using subfolder 'openstack' anymore

### CODE REVIEW

File: test/requests/image/image_tests.rb [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    it "#list_public_images" do
+      Fog::Image[:openstack].list_public_images.body.must_match_schema('images' => [@image_format])
+    end
+
+    it "#list_public_images_detailed" do
+      Fog::Image[:openstack].list_public_images_detailed.body.
+        must_match_schema('images' => [@detailed_image_format])
+    end
+
+    before do
```
Comment (petrblaho) [inline test/requests/image/image_tests.rb]: I would move this in the beginning of `describe` block.

File: test/requests/image/image_tests.rb [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      end
+
+      @instance = Fog::Image[:openstack].create_image(image_attributes).body
+    end
+
+    it "#create_image" do
+      @instance.must_match_schema('image' => @detailed_image_format)
+    end
+
+    after do
```
Comment (petrblaho) [inline test/requests/image/image_tests.rb]: The same as with `before` block above.

**Comment by petrblaho** [comment]
Apart from two `before`/`after` blocks location (see inline notes) looks good to me.

File: test/requests/image/image_tests.rb (hunk 1 above)
Comment (gildub) [inline test/requests/image/image_tests.rb]: Done. Thanks

File: test/requests/image/image_tests.rb (hunk 2 above)
Comment (gildub) [inline test/requests/image/image_tests.rb]: Done. Thanks

**Pull request merged**